services/backend/src/routes/connectivity.js

```diff
@@ -114,6 +114,43 @@ module.exports = (errorHandler, connectivityService) => {
       });
   }));
 
+
+  /**
+   * @apiGroup s_pa_connectivity
+   * @api {get} /connectivity/timeline TimelineByCategory
+   * @apiName TimelineByCategory
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Values of protected and protected connected of all years for a given area
```

Comment:
Please improve this description i think you men "values of protected areas and connected protected areas..."

Reply:
I would suggest a better description: "Values through time of a protected area connectivity category in a given area"

Reply:
I think "Values through time of a protected area connectivity category in a given area" adjusts better to the name TimelineByCategory we chose.